src/main/java/com/maxmind/minfraud/response/Device.java

```diff
@@ -0,0 +1,35 @@
+package com.maxmind.minfraud.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * This class contains minFraud response data related to the device.
+ */
+public final class Device {
+    private final String id;
+
+    public Device(
+            @JsonProperty("id") String id
+    ) {
+        this.id = id;
+    }
+
+    public Device() {
+        this(null);
+    }
+
+    /**
+     * @return The device id.
+     */
+    public String getId() {
```

Comment:
I wonder if we should make this an actual `UUID` object (as well as minFraud ID). That's essentially what I did in the .NET reader.